_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,54 @@
+
+With nearly 2000 exoplanets discovered [so far](http://planetquest.jpl.nasa.gov/newworldsatlas) &mdash; of which 40-odd are [potentially habitable](https://en.wikipedia.org/wiki/List_of_potentially_habitable_exoplanets) &mdash; Earth may soon no longer be unique in its ability to harbor life.
+One signal that we have been listening for in the search for extraterrestrial intelligence is radio waves, which our rapidly industrialized Earth civilization has also been beaming into the ether for the past 100 years or so.
+These artificial signals traveling at the speed of light were humanity's first "hello" to the universe.
+But even these signals are becoming silenced with the end of terrestrial broadcasting; if the same thing happened to E.T., what are some other large-scale signs that could indicate the presence of intelligent life?
+
+The thermal radiation of a planet may be an unintentional calling card, a biomarker of extraterrestrial life.
+In particular, the way heat is distributed as islands in space and time on a planet's surface &mdash; think of the bright spots of illuminated cities on Earth at night &mdash; may be a signature of Earth-like civilizations.
+Because the trend on this planet is towards ever greater energy consumption, civilizations as or more advanced than Earth's may also recognized by the heat (that is, infrared light as seen by our telescopes) that accompanies energy production.
+
+![Kepler-16b, an exoplanet orbiting a pair of stars]({{ site.url }}/images/2015-05-30-Kepler_16b.jpg)
+
+> _Kepler-16b, an exoplanet orbiting a pair of stars. Modified from original [image](http://planetquest.jpl.nasa.gov/system/secondary_files/binaries/988/original/Kepler_16b_39x27.jpg) / Courtesy NASA/JPL-Caltech / Public Domain_
+
+The researchers modeled what heat signals might come off an exoplanet, using Earth as a basis.
+Weighing the energy inputs (solar and planetary heat sources) and outputs (photosynthesis and other biological activity as well as technology) for their model planet, they reached the conclusion that as you produce more energy that is returned to the planet as heat, the model planet starts to warm up.
+As the authors put it, "a civilization with growing power needs will eventually reach a point where they become uncomfortably warm."
+
+Even if the alien civilizations found a way to engineer their planet to reflect heat out of the atmosphere, this would still leave a detectable heat signature.
+The more advanced a civilization is, the argument goes, the more energy it needs, and the more it will move towards harnessing all the available energy in its solar system, perhaps culminating in a [Dyson sphere](http://en.wikipedia.org/wiki/Dyson_sphere).
+We can compare energy consumption with the theoretical maximum available (i.e. the flux of the host star) to get a measure of how advanced (in terms of energy consumption and thus heat output) a civilization is.
```

Comment:
Instead of "flux", say something like "the amount of energy incident from the host star"?